src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -265,6 +265,13 @@ void FullyConnected::getSupportedDescriptors() {
     }
 }
 
+void FullyConnected::createPrimitive() {
+    setPostOps(attr, outDims);
+    attr.set_scratchpad_mode(dnnl::scratchpad_mode::user);
+    Node::createPrimitive();
+    appendPostOpArgs(attr, primArgs, postOpsArgs);
```

Comment:
Is there any reason why `Node::createPrimitive();` is not the last statement, or does `appendPostOpArgs()` depending on something happened inside `Node::createPrimitive();`?

Reply:
Usually filling up the `primArgs` map is the last step of the parameter preparation, and, though is this particular case is does not have any matter, I wanted to preserve thins sequence of actions.

Reply:
personally I prefer to use same mechanism for similar task, for example we can follow what convolution node does inside `prepareParam` instead of introducing `createPrimitive()` overload, this overload becomes tricky due to the timing, and you must be cautious on what shapes are fake/dummy and what information is truly determined.

Reply:
The thing is, in Convolution node we still have a situation when we rebuild `attr` every `preparePrams()` call due to `sum` fusing. But here we always can do it only once. That means this is implementation dependent feature, and given the fact that the node implementation is the most low level, probably we can not do it in a generic fashion. I mean if in the Convolution node we could initialize `attr` only once to in all cases, we probably would do it in the same way as in FC.